google/cloud/forseti/common/gcp_api/errors.py

```diff
@@ -14,9 +14,24 @@
 
 """API errors."""
 
+from google.cloud.forseti.common.util import log_util
+
+LOGGER = log_util.get_logger(__name__)
+
+API_EXECUTION_ERROR_ARG_FORMAT = '{}, {} = {},'
+# {Resource}, {Identifier_key} = {Identifier_value},
+
 
 class Error(Exception):
     """Base Error class."""
+    def __init__(self, message):
+        """Init
+
+        Args:
+            message (str): error message
+        """
+        LOGGER.error(message)
```

Comment:
It would be better for the raiser or the catcher to log the error, rather than log it here where you can't tell where the exception is coming from.

Reply:
updated the logging location